llvm/lib/SYCL/KernelProperties.cpp

```diff
@@ -30,13 +30,17 @@ KernelProperties::KernelProperties(Function &F) {
       continue;
     auto *Alloca =
         dyn_cast_or_null<AllocaInst>(getUnderlyingObject(CB->getOperand(0)));
+    // sycl buffer's property for ddr bank association is lowered
```

Comment:
With the explanation it is quite more palatable, thanks. :-)
But respect the various official spelling:
```suggestion
    // SYCL buffer's property for DDR bank association is lowered
```
By the way it is not only DDR bank in general, but also HBM, etc.

Reply:
Thanks for the typos.

I am aware taht we can also support HBM and that I've made MAXIBundle a struct and not a string so we can add field for memory type, but as of now I think we only have properties for DDR banks.